src/v2/view-builder/views/shared/email.js

```diff
@@ -1,10 +1,6 @@
 import { loc } from 'okta';
 export default {
   title () {
-    if(this.options.currentViewState.name === 'enroll-factor') {
-      return loc('email.enroll.title', 'login');
-    } else {
-      return loc('oie.email.mfa.title', 'login');
-    }
+    return loc('oie.email.mfa.title', 'login');
```

Comment:
Remove the `enroll-factor` instead of replace it to `enroll-authenticator`, in the test it expected to get `loc('oie.email.mfa.title', 'login')` (Verify with your email), instead of `Set up Email Authentication`

Reply:
What about the enroll email flow? @anipendakur-okta would you help to confirm?

Reply:
Yeah, previously it goes with 'Verify with your email' when `enroll-authenticator` with email.

Reply:
`email.enroll.title` was actually `Set up Email Authentication` for enroll flow -
![image](https://user-images.githubusercontent.com/62976351/105112617-467b6780-5a78-11eb-9a73-a536cf5c9ffa.png) (I don't see a test for it).

Looks like this changes it to `Verify with your email`. Is this intended?

Reply:
@anipendakur-okta @haishengwu-okta Interesting, I saw the test for enroll-email is check the `Verify with your email`, code: https://github.com/okta/okta-signin-widget/blob/master/test/testcafe/spec/EnrollAuthenticatorEmail_spec.js#L56

And here is what I get from the playground without my change:
![image](https://user-images.githubusercontent.com/47287459/105115331-6e20fe80-5a7d-11eb-90a0-9536b7b3d631.png)

Reply:
I am guessing that no one explicitly enrolled email and hence dint run into this issue ? interestingly there was no selenium test failures for this I guess. We need to close the loop on this real quick to move forward.  @anipendakur-okta  I guess we can leave the change in this file as is and you can create a jira to fix this ?